src/components/forms/application/v2/type-of-funding.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import { useApplicationStep } from '@/contexts/';
+import ApplicationStepWrapper from '@/components/forms/application/v2/ApplicationStepWrapper';
+import CardWithRadio from '../../fields/CardWithRadio';
+import { FormData } from '@/types';
+
+const fundingPurposeOptions = [
+  { value: 'A business loan', label: 'A business loan' },
+  { value: 'A line of credit', label: 'A line of credit' },
+  { value: 'Both', label: 'Both' },
+];
+
+const TypeOfFunding = () => {
+  const { formData, saveFormData, isStepCompleted, moveForward, isNavigating } =
+    useApplicationStep('type-of-funding');
+
+  const [localFormData, setLocalFormData] = useState<FormData>({
+    typeOfFunding: '',
```

Comment:
This is not the place in which we're storing the type of funding. Should be `loanType`, according to [this](https://github.com/try-keep/referral-capital-application/blob/bd312389f146c7000efdb94d3a66718c1a8ac72b/src/app/step/%5Bid%5D/page.tsx)
```suggestion
    loanType: '',
```